source/extensions/filters/http/json_to_metadata/filter.h

```diff
@@ -76,6 +77,8 @@ class FilterConfig {
   Rules generateRules(const ProtobufRepeatedRule& proto_rule) const;
   absl::flat_hash_set<std::string> generateAllowContentTypes(
       const Protobuf::RepeatedPtrField<std::string>& proto_allow_content_types) const;
+  Regex::CompiledMatcherPtr generateAllowContentTypeRegexs(
```

Comment:
This doesn't use any of the members of this object and isn't used elsewhere wdyt of just pushing this as a free function in the anonymous namespace in the filter.cc since this is just an implementation detail

Reply:
@JuniorHsu  looks like generateAllowContentTypes is used in filter.cc also, should we make both of them a free function in the anonymous namespace?

Reply:
yea I think it it'd work for both of those

Reply:
sure let's do it thanks